Dockerfile.dapper

```diff
@@ -109,7 +112,7 @@ ENV DAPPER_ENV REPO TAG DRONE_TAG DRONE_COMMIT DRONE_BRANCH DRONE_BUILD_EVENT SY
 ENV DAPPER_SOURCE /go/src/github.com/rancher/rancher/
 ENV DAPPER_OUTPUT ./bin ./dist ./go.mod ./go.sum ./pkg/apis/go.mod ./pkg/apis/go.sum ./pkg/client/go.mod ./pkg/client/go.sum ./scripts/package ./pkg/settings/setting.go ./package/Dockerfile ./Dockerfile.dapper
 ENV DAPPER_DOCKER_SOCKET true
-ENV DAPPER_RUN_ARGS "-v rancher2-go16-pkg-1:/go/pkg -v rancher2-go16-cache-1:/root/.cache/go-build --privileged"
+ENV DAPPER_RUN_ARGS "-v rancher2-go16-pkg-1:/go/pkg -v rancher2-go16-cache-1:/root/.cache/go-build --privileged --network=host"
```

Comment:
If this is the only way to do it, I guess we have to go with it. The runners are protected by a security group so it won't be accessible from the outside. How does this run locally?

Reply:
This runs locally no problem without the need of this. This is only the case when running in a DinD environment, simply because the k3d cluster container needs to be able to communicate with its host container in order to be imported to rancher. I can run it outside of a docker container.